addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10976,11 +10976,11 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#20369"
-msgid "Show plot for unwatched items"
+msgid "Show plot for unwatched items (Hides plot and thumb when disabled)"
```

Comment:
>Actually this label also applies for movies

"Show plot and episode thumbnail for unwatched items"?

In the description clarify that plot applies to both movies and episodes.